cartridges/bm_adyen/site-template/meta/system-objecttype-extensions.xml

```diff
@@ -379,6 +387,7 @@
                 <attribute attribute-id="Adyen_HMACkey"/>          
                 <attribute attribute-id="Adyen_Mode"/>
                 <attribute attribute-id="Adyen_Debug"/>
+                <attribute attribute-id="AdyenShipBeforeDate"/>
```

Comment:
Could you add in the configuration the value 5 as default so it is always filled in and no error in request are made